velociraptor/catalogue/registration.py

```diff
@@ -1522,6 +1522,29 @@ def registration_element_masses_in_stars(
 
     return
 
+def registration_snia_rates(
+    field_path: str, unit_system: VelociraptorUnits
+) -> (unyt.Unit, str, str):
+    """
+    Registers the SNIa rates within apertures
+    """
+
+    unit = unit_system.velocity/unit_system.length
+    # Capture aperture size
+    match_string = "Aperture_SNIaRates_aperture_total_star_([0-9]*)_kpc"
+    regex = cached_regex(match_string)
+
+    match = regex.match(field_path)
+
+    if match:
+        aperture_size = match.group(1)
+
+        full_name = f"SNIa rate ({aperture_size} [# $\\rm yr^{-1}$])"
```

Comment:
You should not provide a unit here, since that depends on the units the user chooses. The default unit is `velocity unit/length unit`, which is not `yr^-1`.